schemas/device/electrodeUsage.schema.tpl.json

```diff
@@ -1,59 +1,30 @@
 {
   "_type": "https://openminds.ebrains.eu/ephys/ElectrodeUsage",
-  "_categories": [
-    "deviceUsage"
-  ],
-  "required": [
-    "electrode"
-  ],
+  "_extends": "/core/schemas/research/deviceUsage.schema.tpl.json",
   "properties": {
-    "additionalInformation": {
-      "type": "array",
-      "minItems": 1,
-      "uniqueItems": true,
-      "_instruction": "Add one or several files containing any additional information about this the electrode.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/core/File",
-        "https://openminds.ebrains.eu/core/FileBundle"
-      ]
-    },
     "anatomicalLocation": {
-      "_instruction": "Add the anatomical entity in which the electrode contact is located.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-        "https://openminds.ebrains.eu/sands/CustomAnatomicalEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntityVersion"
+      "_instruction": "Add the anatomical entity that describes the semantic location of the electrode contact best.",
```

Comment:
"Add the anatomical entity that semantically best describes the anatomical location of the electrode contact."